__tests__/app.test.js

```diff
@@ -49,3 +49,41 @@ describe("Route not found", () => {
       });
   });
 });
+
+describe("/api/articles/:article_id", () => {
+  test("200: responds with a single article chosen by Id number", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
```

Comment:
I think expecting body.article toMatchObject would have been a little easier here but what you've done works too :)